scheduler.py

```diff
@@ -1,128 +0,0 @@
-import time
-from datetime import datetime
-
-import pymongo
-from pymongo import MongoClient
-
-import praw
-
-import yfinance as yf
-from dotenv import load_dotenv
-
-import dotenv
-import os
-
-
-# Reads, parses and converts the ticker files into dictionary
-def build_dictionary(file_name, ticker_dict):
```

Comment:
`def build_dictionary(file_name, ticker_dict):
    with open(file_name, 'r') as file:
        for line in file:
            line_list = line.strip().split("\t")
            if len(line_list) < 2:
                continue

            symbol = line_list[0]
            name = line_list[1]
            if symbol not in ticker_dict:
                    ticker_dict[symbol] = name`